projects/ngu-carousel/src/lib/ngu-carousel/ngu-carousel.component.ts

```diff
@@ -59,12 +59,11 @@ export class NguCarousel<T> extends NguCarouselStore
   private withAnim = true;
   activePoint: number;
   isHovered = false;
-
-  @Input() private inputs: NguCarouselConfig;
-  @Output() private carouselLoad = new EventEmitter();
-
+  @Input() inputs: NguCarouselConfig;
+  @Output() carouselLoad = new EventEmitter();
+  // tslint:disable-next-line:no-output-on-prefix
   // tslint:disable-next-line:no-output-on-prefix
```

Comment:
Thank you @amine-bambrik-p8 just one small change then we can merge this comment is twice here.